src/main.rs

```diff
@@ -145,11 +152,17 @@ The second SOURCE represents the well-known Kafka "offset" property. Kafka Delta
 
             let app_id = ingest_matches.value_of("APP_ID").unwrap().to_string();
 
-            let starting_offsets = ingest_matches
-                .value_of("STARTING_OFFSETS")
-                .unwrap()
-                .to_string();
-            let starting_offsets = StartingOffsets::from_string(starting_offsets)?;
+            let seek_offsets: Option<Vec<(DataTypePartition, DataTypeOffset)>> = ingest_matches
+                .value_of("SEEK_OFFSETS")
+                .map(|s| serde_json::from_str(s).expect("Cannot parse seek offsets"));
```

Comment:
Nooiice. I like the simplification we get by splitting seek offsets from the auto offset reset config and explicitly _not_ supporting the more generic `starting_offsets` concept.

Reply:
yeah, it makes more sense, there's no starting_offsets option for the kafka anyways